sim_pipeline/sim_lsst_galaxy_galaxy_lens_simulation.py

```diff
@@ -0,0 +1,283 @@
+import os
+import random
+# lenstronomy module import
+import lenstronomy
+import matplotlib.pyplot as plt
+import numpy as np
+import skypy
+from lenstronomy.SimulationAPI.sim_api import SimAPI
+from lenstronomy.Util import constants
+from lenstronomy.Cosmo.lens_cosmo import LensCosmo
+from skypy.pipeline import Pipeline
+from skypy.galaxies.velocity_dispersion import schechter_vdf
+from astropy.cosmology import FlatLambdaCDM
+from astropy.visualization import make_lupton_rgb
+from lenstronomy.SimulationAPI.ObservationConfig.LSST import LSST
+from lenstronomy.ImSim.image_model import ImageModel
+cosmo = FlatLambdaCDM(H0=67.66, Om0=0.30966)
```

Comment:
I would split the routines and the specification of the cosmology, such that the user can chose them at input without changing the source code of this file